### ISSUE TITLE
The data division? 

### ISSUE THREAD

**Issue by ghost** [NjtechCVLab/RSTPReid-Dataset#1]
In paper, there are 3701, 200 and 200 identities are utilized for training, validation and testing.
Can you provide the data division file?
The IDs for training, validation and testing.
Thanks.

**Comment by Suo-Wei** [comment]
Same question。。。

**Comment by ZJWang9928** [comment]
Thanks for your interest in our work.
You can just simply divide the data as the training set, validation set and testing set by ID in order.
